test/OpenTelemetry.Exporter.Geneva.Tests/GenevaLogExporterTests.cs

```diff
@@ -924,5 +928,10 @@ private void AssertFluentdForwardModeForLogRecord(GenevaExporterOptions exporter
             // Epilouge
             Assert.Equal("DateTime", timeFormat["TimeFormat"]);
         }
+
+        // A custom exception class with non-ASCII character in the type name
+        private class CustomExceptionû : Exception
```

Comment:
Consider a different approach (e.g. reflection, dynamic types) to avoid non-ASCII characters in the source code.

Reply:
I replaced the character by its code. This should be good enough?

Reply:
Thanks!